src/pages/getting-started/app-guides/app-authorization/index.mdx

```diff
@@ -7,12 +7,30 @@ tagline: "Find out how to implement an authorization flow and configure the App
 desc: "Find out how to implement an authorization flow and configure the App authorization building block."
 ---
 
+# Authorization for LiveChat apps
+
 If you want to build an integration that calls the LiveChat APIs, you'll need to implement an authorization flow and configure the **App authorization** building block.
 
 - <SectionLink to={"/authorization/authorizing-api-calls/"}>Read about different authorization flows</SectionLink>
 
 - <SectionLink to={"/authorization/authorization-in-practice/"}>Tutorial: Authorization in practice </SectionLink>
 
 <div style={{maxWidth: '600px'}}>
```

Comment:
przez maxwidth to zdjęcie się pikseluje :D dałabym może 200?

Reply:
Ja właśnie nie ogarniam jeszcze czasem tych zdjęć xd Zmieniam na 200 i zobaczę jak będzie wyglądać :)